graphql/tests/mod_test.ts

```diff
@@ -0,0 +1,101 @@
+import { Rhum } from "../../test_deps.ts";
+import { GraphQL } from "../mod.ts";
+import { buildSchema } from "./deps.ts";
+import { Drash } from "../../deps.ts";
+
+//
+// DATA
+//
+const schema = buildSchema(`
+  type Query {
+    hello: String
+  }
+`);
+const root = {
+  hello: () => {
+    return "Hello world!";
+  },
+};
+const graphQL = GraphQL({ schema, graphiql: true, rootValue: root });
+class GraphQLResource extends Drash.Http.Resource {
+  static paths = ["/graphql"];
+
+  @Drash.Http.Middleware({
+    before_request: [graphQL],
+  })
+  // Used purely as an endpoint for the frontend playground
+  public GET() {
+    return this.response;
+  }
+
+  // Used purely as an endpoint to make a query from the client
+  @Drash.Http.Middleware({
+    before_request: [graphQL],
+    after_request: [],
+  })
+  public POST() {
+    return this.response;
+  }
+}
+const server = new Drash.Http.Server({
+  resources: [GraphQLResource],
+});
+async function serverAction(action: "run" | "close") {
```

Comment:
oh i like this function. wonder if it'll be helpful in drash TestHelper for others... 🤔 @crookse

Reply:
yea. we have something that helps us start and stop servers currently but not as easy as this

Reply:
i think i've done something similar in various repos lol, never thought of it being a function **all** modules/repos import

Reply:
I think the only bit worth abstracting is just that 8 (or so) line function, because we may as well just create the server instance like above, so i don't think creating a single point is going to make much difference?

And just to clarify, are we on about turning this logic into some method we place into the services repo, then our other repos just import that?